### PR TITLE
[DO NOT MERGE YET] Making deathsquad relevant again

### PR DESCRIPTION
**Pull request by Duny-**
- Added deathsquad magboots, featuring no slowdown and emo colors.
- Moved deathsquad helmet to
/obj/item/clothing/head/helmet/space/rig/deathsquad.
- Created /obj/item/clothing/suit/space/rig/deathsquad. Deathsquad
members used to wear some shitty swat armor instead of proper space rig.
- Tweaked armor values for both to be overall stronger, and not weaker,
than the station's stuff. Both now have atmos suit level fire protection
and the suit can now hold more useful items.
- Deathsquad members no longer spawn with a flashlight, their helmet now
has a built-in light.
- Their gloves are now insulated.
- They no longer spawn with any flashbangs.
- They now spawn with an energy shield.
- They now all have a pinpointer. The leader is still the only one with
a disk.
- They now spawn with a double extended emergency oxygen tank instead of
the other shitty one.

DO NOT MERGE YET I have sprite issues to fix

### CODE REVIEW

File: code/modules/clothing/suits/armor.dm [hunk 1]
```diff
@@ -84,24 +84,6 @@
 	armor = list(melee = 10, bullet = 10, laser = 80, energy = 50, bomb = 0, bio = 0, rad = 0)
 	siemens_coefficient = 0
 
-/obj/item/clothing/suit/armor/swat
```
Comment (ComicIronic) [inline code/modules/clothing/suits/armor.dm]: Why remove this?
And why remove it and not remove the one underneath?

**Comment by Boxe** [comment]
THANK YOU DUNY

File: code/modules/clothing/suits/armor.dm (hunk 1 above)
Comment (Duny-) [inline code/modules/clothing/suits/armor.dm]: It's the former path to the snowflake armor deathsquid were using. I didn't notice the one underneath but it sounds even more snowflakey, I'm deleting that too.

**Comment by Duny-** [comment]
NOT CLEAN ENOUGH, REOPENING A NEW ONE

**Pull request closed without merging**